doc/lightning-waitsendpay.7.txt

```diff
@@ -8,7 +8,7 @@ lightning-waitsendpay - Protocol for sending a payment via a route.
 
```

Comment:
Hmm, just noticed this title line is wrong: cut & paste bug?

Reply:
Well, it is part of the protocol for paying via a route.  You perform `sendpay`, then when convenient you perform a `waitsendpay` to monitor the status of the payment.